tools/Customization/DevHome.Customization/Views/DevDriveInsightsView.xaml

```diff
@@ -30,20 +30,20 @@
                     </ctControls:SettingsCard.HeaderIcon>
                     <Grid Grid.Column="2">
                         <Grid.RowDefinitions>
-                            <RowDefinition Height="Auto"  />
+                            <RowDefinition Height="Auto" />
                             <RowDefinition Height="Auto" />
                             <RowDefinition Height="Auto" />
                         </Grid.RowDefinitions>
-                        <TextBlock Style="{StaticResource CardBodyStackPanelAltTextBlockStyle}" HorizontalAlignment="Right" Margin="0 0 115 5">
-                                <Run Text="{Binding Path=DevDriveSizeText, Mode=OneWay}" />
+                        <TextBlock Style="{StaticResource CardBodyStackPanelAltTextBlockStyle}" HorizontalAlignment="Left" Margin="0 0 115 5">

```

Comment:
For these TextBlocks, why are you binding to the Run rather than the Text on the TextBlock itself?

Reply:
For posterity.  Mixing styles with localized strings does not mix well with translation.  It needs to be avoided.